pkgs/by-name/ni/nixos-container/package.nix

```diff
@@ -15,8 +16,9 @@ replaceVarsWith {
   src = ./nixos-container.pl;
 
   replacements = {
-    perl = perl.withPackages (p: [ p.FileSlurp ]);
+    perl = perl.withPackages (p: [ p.FileSlurp p.IPCRun ]);
     su = "${shadow.su}/bin/su";
+    lib = "${path + "/lib"}";
```

Comment:
I know that this kinda sucks, open for better ideas (I need `versionAtLeast` since Perl's version libraries fail to parse stuff like `25.11pre-git`).